extensions/amp-pinterest/0.1/test/test-amp-pinterest.js

```diff
@@ -57,4 +78,38 @@ describes.realWin('amp-pinterest', {
_name': 'kentbrew', 'provider_favicon_url': 'https://s.pinimg.com/images/api/attrib/flickr.png', 'author_url': 'http://www.flickr.com/photos/kentbrew/', 'provider_name': 'flickr'}, 'description': 'Love my Pinterest t-shirt!', 'pinner': {'about': "Widgeteer, Pinterest. If the Pin It button is broken or confusing, it's my fault.", 'location': 'Runnymede Poultry Colony', 'full_name': 'Kent Brewster', 'follower_count': 4292, 'image_small_url': 'https://i.pinimg.com/30x30_RS/dc/24/76/dc2476722ee6e8c12b6a93d8ac5b6a4d.jpg', 'pin_count': 5068, 'id': '99360872937292935', 'profile_url': 'https://www.pinterest.com/kentbrew/'}, 'repin_count': 3444, 'aggregated_pin_data': {'aggregated_stats': {'saves': 109743, 'done': 90}}, 'rich_metadata': {'aggregated_app_link': {'ipad': null, 'android': null, 'ios': {'app_url': 'flickr://flickr.com/photos/kentbrew/6851755809', 'app_name': 'Flickr', 'app_id': '328407587'}, 'iphone': {'app_url': 'flickr://flickr.com/photos/kentbrew/6851755809/', 'app_name': 'Flickr', 'app_id': '328407587'}}, 'has_price_drop': false, 'site_name': 'Flickr', 'description': 'Today was my last day at Lexity. Next up?  A little start-up in Palo Alto.', 'link_status': 0, 'title': 'Next stop: Pinterest!', 'locale': 'en', 'amp_valid': false, 'amp_url': '', 'url': 'http://www.flickr.com/photos/kentbrew/6851755809/', 'tracker': null, 'apple_touch_icon_link': 'https://s-media-cache-ak0.pinimg.com/favicons/b4ef6053716205a6beab42b56030364fc1e6dce4e62e863b3e4adb0e.png?a4077badc755ea44f96490b95c0278a4', 'favicon_images': {'orig': 'https://s-media-cache-ak0.pinimg.com/favicons/f1def2d953077deabc5605a6c5f00f6f361a2c491b0ec4bd08f77659.ico?d0236f158c380b9595ffa7c7d7d1a433'}, 'article': {'description': 'Today was my last day at Lexity. Next up? A little start-up in Palo Alto.', 'date_published': null, 'authors': [], 'type': 'articlemetadata', 'id': '140248253674512', 'name': 'Next stop: Pinterest!'}, 'type': 'richpindataview', 'id': '811ea1aef1bcaf6342a725365871d79d', 'apple_touch_icon_images': {'orig': 'https://s-media-cache-ak0.pinimg.com/favicons/b4ef6053716205a6beab42b56030364fc1e6dce4e62e863b3e4adb0e.png?a4077badc755ea44f96490b95c0278a4'}, 'favicon_link': 'https://s-media-cache-ak0.pinimg.com/favicons/f1def2d953077deabc5605a6c5f00f6f361a2c491b0ec4bd08f77659.ico?d0236f158c380b9595ffa7c7d7d1a433'}, 'dominant_color': '#e77b60', 'link': 'http://www.flickr.com/photos/kentbrew/6851755809/', 'board': {'description': "Some of the fun we're having bringing Pinterest to life!", 'url': '/kentbrew/adventures-at-pinterest/', 'follower_count': 1128, 'image_thumbnail_url': 'https://s-media-cache-ak0.pinimg.com/upload/99360804217844427_board_thumbnail_2017-07-07-20-33-41_48158_60.jpg', 'pin_count': 151, 'id': '99360804217844427', 'name': 'Adventures at Pinterest'}, 'videos': null, 'images': {'237x': {'url': 'https://i.pinimg.com/237x/a7/66/56/a76656e966b1958f568d63c3f1c05aec.jpg', 'width': 237, 'height': 177}}, 'embed': null, 'is_video': false, 'id': '99360735500167749'}]}
```

Comment:
I am going to store these as consts. 

Or I can remove the data and just keep the data needed around.

Reply:
it's fine IMO.

Reply:
Ack. Retained it as such. For my knowledge, this exceeds 80 characters but gulp lint didn't raise issues. Does it ignore this?